updates/update_settings_to660.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace GinoPane\AwesomeIconsList\Updates;
+
+use DB;
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+/**
+ * Class UpdateSettingsTo590
+ *
+ * @package GinoPane\AwesomeIconsList\Updates
+ */
+class UpdateSettingsTo660 extends UpdateSettingsAbstract
+{
+    protected static $PREVIOUS_FONTAWESOME_LINK = 'https://use.fontawesome.com/releases/v5.12.0/css/all.css';
+    protected static $PREVIOUS_FONTAWESOME_LINK_ATTRIBUTES = [
```

Comment:
Settings for 5.12 were without integrity attribute:

![image](https://github.com/user-attachments/assets/6e0da1c5-fb2a-43bd-9157-96ed2bb789c2)

It might cause issues on rollback to a previous version. Please adjust